### ISSUE TITLE
toLower() and toUpper() do not change the case of non-English letters

### ISSUE THREAD

**Issue by LiorKogan** [RedisGraph/RedisGraph#2670]
Greek:

```
GRAPH.QUERY g "RETURN toLower('ΑαΒβΓγΔδΕεΖζΗηΘθΙιΚκΛλΜμΝνΞξΟοΠπΡρΣσςΤτΥυΦφΧχΨψΩω')"
```
Expected: "ααββγγδδεεζζηηθθιικκλλμμννξξοοππρρσσςττυυφφχχψψωω"

```
GRAPH.QUERY g "RETURN toUpper('ΑαΒβΓγΔδΕεΖζΗηΘθΙιΚκΛλΜμΝνΞξΟοΠπΡρΣσςΤτΥυΦφΧχΨψΩω')"
```
Expected: "ΑΑΒΒΓΓΔΔΕΕΖΖΗΗΘΘΙΙΚΚΛΛΜΜΝΝΞΞΟΟΠΠΡΡΣΣΣΤΤΥΥΦΦΧΧΨΨΩΩ"

Cyrillic:

```
GRAPH.QUERY g "RETURN toLower('АаБбВвГгДдЕеЖжЗзИиЙйКкЛлМмНнОоПпРрСсТтУуФфХхЦцЧчШшЩщЬьЭэЮюЯя')
```
Expected: "ааббввггддеежжззииййккллммннооппррссттууффххццччшшщщььээююяя"

```
GRAPH.QUERY g "RETURN toUpper('АаБбВвГгДдЕеЖжЗзИиЙйКкЛлМмНнОоПпРрСсТтУуФфХхЦцЧчШшЩщЬьЭэЮюЯя')
```
Expected: "ААББВВГГДДЕЕЖЖЗЗИИЙЙККЛЛММННООППРРССТТУУФФХХЦЦЧЧШШЩЩЬЬЭЭЮЮЯЯ"

but currently, no conversion is done.

**Comment by LiorKogan** [comment]
Case mapping should follow the mapping defined in UnicodeData.txt (available in the source code of the [ICU 72.1](https://github.com/unicode-org/icu/releases/tag/release-72-1)); [format](https://www.unicode.org/L2/L1999/UnicodeData.html), [more details](https://unicode.org/faq/casemap_charprop.html)

**Comment by gitgudyyao** [comment]
this issue is that the toLower and toUpper functions in AGE do not support the Greek and Cyrillic characters you are using. 

These functions only support ASCII characters, so they may not work correctly with non-ASCII characters like Greek and Cyrillic. 

To convert Greek and Cyrillic characters to lowercase or uppercase, you can use a different method that is designed to handle non-ASCII characters. One option is to use the toLowerCase and toUpperCase methods provided by the JavaScript String object. 

For example: 
GRAPH.QUERY g "RETURN 'ΑαΒβΓγΔδΕεΖζΗηΘθΙιΚκΛλΜμΝνΞξΟοΠπΡρΣσςΤτΥυΦφΧχΨψΩω'.toLowerCase()" 

"ααββγγδδεεζζηηθθιικκλλμμννξξοοππρρσσςττυυφφχχψψωω" GRAPH.QUERY g "RETURN 'АаБбВвГгДдЕеЖжЗзИиЙ
